tests/manage/noobaa/test_basic_bucket_operations.py

```diff
@@ -0,0 +1,73 @@
+import logging
+import pytest
+
+from tests.helpers import create_unique_resource_name
+from ocs_ci.framework import config
+from ocs_ci.framework.pytest_customization.marks import tier1
+from ocs_ci.ocs.resources import noobaa
+
+
+logger = logging.getLogger(__name__)
+
+
+@pytest.fixture()
+def noobaa_obj():
```

Comment:
we already have this fixture in tests/manage/noobaa/test_create_bucket.py
Please create a conftest in the noobaa dir hierarchy and move this fixture to there

Reply:
sure